nix/modules/flake-parts/haskell.nix

```diff
@@ -100,6 +108,44 @@
               };
               is_tty = true;
             };
+          setup = {
+            command = pkgs.writeShellApplication {
+              name = "vira-setup";
+              runtimeInputs = [ pkgs.curl ];
```

Comment:
Could you add `meta.decription`? What's the intent of this script? Does it forcefully inject these repos even if the user already has added theirs in the settings?

Reply:
> Does it forcefully inject these repos even if the user already has added theirs in the settings?

Yes, if the repos don’t already exist.

Reply:
Okay, that's fine. It is for development only anyway.